src/plugins/folderView/components/ListView.vue

```diff
@@ -170,6 +170,11 @@ export default {
             } else if (this.sortClass === 'desc') {
                 return this.items.sort(this.descending.bind(this));
             }
+        },
+        formatTime () {
```

Comment:
I don't quite understand how this still works, since the invocation hasn't changed?

Reply:
computed items are invoked every time a property changes, thus formatTime would not be a function, so I had to wrap as a closure to allow me to pass arguments to the function